WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -129,48 +121,40 @@ private extension HubMenu {
             .foregroundColor(Color(menu.iconColor))
         }
         .accessibilityIdentifier(menu.accessibilityIdentifier)
-        .overlay {
-            if menu.id != HubMenuViewModel.GoogleAds.id {
-                NavigationLink(value: menu.id) {
-                    EmptyView()
-                }
-                .opacity(0)
-            }
-        }
     }
 
     @ViewBuilder
-    func detailView(menuID: String) -> some View {
+    func detailView(destination: HubMenuNavigationDestination) -> some View {
         Group {
-            switch menuID {
-            case HubMenuViewModel.Settings.id:
+            switch destination {
+            case .payments:
+                    paymentsView
+            case .settings:
                 SettingsView()
                     .navigationTitle(HubMenuViewModel.Localization.settings)
```

Comment:
Let's keep these in the original order, it matches how they're displayed.

Reply:
Done in ba0e069a3c3ea7c640338a4927d63bafd076c8ed.